src/scss/mixins/_buttons.scss

```diff
@@ -24,6 +24,12 @@
     border-color: $hover-border;
   }
 
+  &:focus,
```

Comment:
We actually disabled the focus button style in `src/scss/_variables.scss` ... I wonder if you could just remove that line and use the default?

Reply:
Well, I tried but we disabled variables linked to it ( you know how, in bootstrap config, variables references other variables and so on). Especially input focus variables. And I don't think we want to reactivate these. So I tried to bypass by using the variables normally assigned to the input and assign them only to the focus btn. But it's ended not working and I thought I might have been over-complicating it and went for a simpler solution. 
I'll try again this afternoon to reactivate them

Reply:
Ok I agree we shouldn't change the input focus but you should be able to do it by changing the `$btn-focus-box-shadow` and `$btn-active-box-shadow` variables that we currently have set to `none`.  Using the variables that Bootstrap provides is the preferred way to customise.